scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -363,8 +364,11 @@ def parse_arguments(args):
                         help="Specify a file where to save logs.")
 
     parser.add_argument(
-        "-M", "--runtime-artifact-cleanup", action="store_true",
-        help="Delete artifacts of passing tests.")
+        "-M", "--runtime-artifact-cleanup", choices=['passing-tests', 'all-tests'],
+        default=None, const='passing-tests', nargs='?',
```

Comment:
Dont we need a default which is compatible with current behavior, ie.. default should be 'pass', how is the default being set? using const?